### ISSUE TITLE
[Quest] Revenge for the Vargul

### ISSUE THREAD

**Issue by koray053** [dalaranwow/dalaran-wow#4813]
Current behaviour: Everything works decent until you need to fight the npc's in order when you have killed vorgul the next npc should appear brita which is not happening she isn't doing her part as she's just standing still in the corner for some reason
Steps to reproduce the problem: Click on the bethods sword at the middle of The Cirlce of Suffering and try the challange yourself you will see that it will not go any further after you have defeated vorgul
Include proofs for this behaviour

Quest: http://www.wowhead.com/quest=13059/revenge-for-the-vargul
Quest Item: http://www.wowhead.com/item=42928/bethods-sword
Npc/Creature: http://www.wowhead.com/npc=30483/volgur
Npc/ Creature: http://www.wowhead.com/npc=30484/brita
Npc/Creature: http://www.wowhead.com/npc=30475/thane-illskar

<a href="http://nl.tinypic.com?ref=34tbs51" target="_blank"><img src="http://i63.tinypic.com/34tbs51.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

**Comment by koray053** [comment]
EDIT: I have done the quest it worked after 4 times of trying the quest again.. and again..

But there is a bug if you're out of a certain distance while Thane llskar is yelling to the next npc that shall challange you the npc will not fight you for some reason and stands just at the place he spawned

**Comment by Rushor** [comment]
-- Duplicate Issue --
Your issue posted here is a duplicate of issue: https://github.com/dalaranwow/dalaran-wow/issues/2628 (< click on this and you will be redirected to the original posted issue). 

As you can see the issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
